### PR TITLE
Changed external NB port for all-in-one env from 8088 to 8080

### PR DESCRIPTION
**Pull request by PinGwynn**

### CODE REVIEW

File: templates/defaults/main.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ topology_engine_endpoint: "http://127.0.0.1"
 topology_engine_rest_endpoint: "http://127.0.0.1"
 topology_engine_rest_port: "80"
 northbound_endpoint: "http://127.0.0.1"
```
Comment (gauravchug) [inline templates/defaults/main.yaml]: Change `127.0.0.1` to `northbound.pendev`

File: templates/defaults/main.yaml [hunk 2]
```diff
@@ -22,8 +22,8 @@ mininet_rest_port: "38080"
 topology_engine_endpoint: "http://127.0.0.1"
 topology_engine_rest_endpoint: "http://127.0.0.1"
 topology_engine_rest_port: "80"
-northbound_endpoint: "http://127.0.0.1"
-northbound_rest_port: "8088"
+northbound_endpoint: "http://northbound.pendev"
```
Comment (abochkarev) [inline templates/defaults/main.yaml]: @PinGwynn Am I correct that 'northbound.pendev' should be resolved in /etc/hosts for the local env? If so, would it be possible to write about in readme file? Also, how about the other endpoints, I mean: openkildagui_host, floodlight_endpoint, mininet_endpoint, topology_engine_endpoint, topology_engine_rest_endpoint. Are you planning to replace them with the DNS?

Comment (PinGwynn) [reply templates/defaults/main.yaml]: No, this endpoint and variable use in kilda gui. But as we have added it under the docker-compose control, we have to change endpoints to "internal" docker-compose names.

Comment (abochkarev) [reply templates/defaults/main.yaml]: @PinGwynn Ok, I got it.

**Pull request merged**